src/components/modal/Modal.scss

```diff
@@ -155,7 +155,9 @@
         }
 
         .modal-close-button {
+            top: initial;
             right: 10px;
+            line-height: 0;
```

Comment:
Out of curiosity, are you testing this property on a specific modal? For general modals, I'm not sure if this would have an effect.